test/models/validator_test.rb

```diff
@@ -198,10 +196,10 @@ class ValidatorTest < ActiveSupport::TestCase
   end
 
   test "#filtered_by multiple parameters excludes correct validators from collection" do
```

Comment:
Same here. Should be something like `#filtered_by multiple parameters includes all validators that meet any of selected criteria"`